examples/step-55/step-55.cc

```diff
@@ -193,18 +193,20 @@ namespace Step55
 
     virtual void vector_value(const Point<dim> &p,
                               Vector<double> &  value) const override;
+
+  private:
+    static constexpr double pi  = numbers::PI;
+    static constexpr double pi2 = numbers::PI * numbers::PI;
```

Comment:
Why not just use the global objects? Introducing aliases seems not worth the two lines.

Reply:
The problem is that we do not have a `pi*pi` as a global constant in [`numbers`](https://www.dealii.org/developer/doxygen/deal.II/namespacenumbers.html).

Shall we introduce another constant `PI_SQ` in `numbers`?